website/client/components/inventory/items/index.vue

```diff
@@ -129,6 +133,18 @@
 
   hatchedPetDialog()
 
+  div.eggInfo(ref="draggingEggInfo")
+    div(v-if="currentDraggingEgg != null")
+      div.potion-icon(:class="'Pet_Egg_'+currentDraggingEgg.key")
+      div.popover
+        div.popover-content {{ $t('dragThisEgg', {eggName: currentDraggingEgg.text }) }}
```

Comment:
could you add to `.popover-content` a `margin: 15px;` its not your fault just would be nice if this is in the same PR :)  

cc @Tressley currently the "dragging" text is completely at the border, so I just think 15px looks good, what do you think ?

Reply:
@negue -- Can you attach a screenshot for reference? 👍

Reply:
@Tressley  
![egg-hatching-drag](https://user-images.githubusercontent.com/842273/34272623-3a28217a-e691-11e7-804b-2d4b73f6d33f.png)

Reply:
Oh, wow. I'm just now seeing this @negue! Sorry for the delay! I think there is some style work needed before merging this. See attached screen below for better direction on positioning and placement.

![image](https://user-images.githubusercontent.com/2686586/34670913-4aeed2c0-f446-11e7-8d68-8b16fad295fb.png)

Reply:
will fix this in a separate PR